joomla/joomla-cms
# [asset_id db field] add default value and normalise across db systems

PR for issue https://github.com/joomla/joomla-cms/issues/13032

### Summary of Changes

As detected in https://github.com/joomla/joomla-cms/issues/13032 there are some issues with the asset_id db columns.
This PR normalizes them.

### Testing Instructions

- On update
  - Run the update queries for your db system manually and check all fine

- On install
  - Use latest staging, apply patch
  - Delete configuration.php file and install as usual. Check all fine and the db fields are correctly created.

### Documentation Changes Required

None.

### Notes

Although the issue was found on 4.0 because of a more strict sql_mode, this also applies in 3.x since this db field should always have a default value.
